examples/player/player.js

```diff
@@ -22,6 +22,7 @@ elms.forEach(function(elm) {
 var Player = function(playlist) {
   this.playlist = playlist;
   this.index = 0;
+  this.step = this.step.bind(this);
```

Comment:
It's pretty common to do that in React components. Not saying it justifies anything but I got used to it and haven't seen any problem.